tests/TestCase/I18n/DateTest.php

```diff
@@ -32,6 +32,28 @@ class DateTest extends TestCase
     protected $locale;
 
     /**
+     * icuVersion
+     *
+     * @var int
+     */
+    protected static $icuVersion;
+
+    /**
+     * setUpBeforeClass
+     *
+     * @return void
+     */
+    public static function setUpBeforeClass()
+    {
+        ob_start();
+        phpinfo();
+        $phpinfo = ob_get_clean();
+        preg_match('/ICU version => ([\d\.]+)/', $phpinfo, $matches);
```

Comment:
Should we do this as part of the test suite `bootstrap.php`? Might be a nice to have a constant with the ICU version.